base/strings/util.jl

```diff
@@ -94,10 +118,10 @@ julia> chomp("Hello\\n")
 """
 function chomp(s::AbstractString)
     i = endof(s)
-    (i < 1 || s[i] != '\n') && (return SubString(s, 1, i))
+    (i < start(s) || s[i] != '\n') && (return SubString(s, start(s), i))
```

Comment:
Better leave this for another PR as it's unrelated. Anyway it's not worth changing these if we don't do it everywhere.

Reply:
ok